packages/mdc-slider/foundation.ts

```diff
@@ -137,15 +147,19 @@ class MDCSliderFoundation extends MDCFoundation {
   init() {
     this.isDiscrete_ = this.adapter_.hasClass(cssClasses.IS_DISCRETE);
     this.hasTrackMarker_ = this.adapter_.hasClass(cssClasses.HAS_TRACK_MARKER);
-    DOWN_EVENTS.forEach((evtName) => this.adapter_.registerInteractionHandler(evtName, this.interactionStartHandler_));
-    this.adapter_.registerInteractionHandler('keydown', this.keydownHandler_);
-    this.adapter_.registerInteractionHandler('focus', this.focusHandler_);
-    this.adapter_.registerInteractionHandler('blur', this.blurHandler_);
+
     DOWN_EVENTS.forEach((evtName) => {
+      this.adapter_.registerInteractionHandler(evtName, this.interactionStartHandler_ as EventListener);
```

Comment:
I think if you change the adapter methods to `SpecificEventListener` you won't need to cast here

Reply:
Done